install/docker/index.markdown

```diff
@@ -31,16 +31,69 @@ And you will likely need to log out and back into your user account for the chan
 
 ## Running MoveIt! Containers
 
-To run a Debian-installed container of MoveIt! with graphics support:
+To use the command line only (no GUI interfaces like Rviz), run this command:
 
-    wget https://raw.githubusercontent.com/ros-planning/moveit/master/.docker/gui-docker -O gui-docker && chmod +x gui-docker
-    ./gui-docker -it --rm moveit/moveit:melodic-release /bin/bash
+    docker run -it moveit/moveit:melodic-release
 
-This will attempt to use nvidia-docker if hardware and drivers are available.
+To use Rviz (and other GUIs), you need to set up hardware acceleration for Docker as described [here](http://wiki.ros.org/docker/Tutorials/Hardware%20Acceleration#nvidia-docker2). For convenience, the instructions for Nvidia GPUs are reproduced below. 
 
-**Command-Line Only**
+First, install nvidia-docker2 according to [these instructions](https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0)). Then, create a directory with a Dockerfile and entrypoint script inside.
 
-    docker run -it moveit/moveit:melodic-release
+    mkdir my_moveit_image & cd my_moveit_image
+    touch Dockerfile
+
+Paste the following content into the Dockerfile. Replace the docker image in the first with the one you want to use.
+
+    FROM moveit/moveit:melodic-release
+
+    # nvidia-container-runtime
+    ENV NVIDIA_VISIBLE_DEVICES \
+        ${NVIDIA_VISIBLE_DEVICES:-all}
+    ENV NVIDIA_DRIVER_CAPABILITIES \
+        ${NVIDIA_DRIVER_CAPABILITIES:+$NVIDIA_DRIVER_CAPABILITIES,}graphics
+
+
+Build the image. Don't forget the period at the end of that command. Then create a script to run the image called `run-gui-moveit-docker`, make it executable, and open it:
```

Comment:
give an example run command, don't assume reader just knows it

Reply:
The four lines below do everything that is described in this line. I guess that wasn't clear.

Reply:
I clarified.